pkg/resource/deploy/step_generator.go

```diff
@@ -1670,7 +1730,11 @@ func (sg *stepGenerator) determineAllowedResourcesToDeleteFromTargets(
 	// Produce a map of targets and their dependents, including explicit and implicit
 	// DAG dependencies, as well as children (transitively).
 	targets := sg.getTargetDependents(targetsOpt)
-	logging.V(7).Infof("Planner was asked to only delete/update '%v'", targetsOpt)
+
+	if !targetsOpt.IsConstrained() {
+		logging.V(7).Infof("Planner was asked to only delete/update '%v'", targetsOpt)
+	}
```

Comment:
Not sure I understand this change.  `!targetOpt.IsConstrained()` is checked for above, and we return if that's the case, so we'll never log anything here now.